index.js

```diff
@@ -151,6 +151,19 @@ MetaInspector.prototype.getMetaDescription = function()
 	return this;
 }
 
+MetaInspector.prototype.elemContainsTag = function(elem,tag) {
+	if (elem.name && elem.name === tag ) {
```

Comment:
Per our Slack convo let's just go ahead and switch to `elem.tagName` since it doesn't break the tests.